R/snapshots.R

```diff
@@ -0,0 +1,116 @@
+#' Run Simulations From Snapshot Files
+#'
+#' @param ... character strings, path to snapshot files or a directory containing snapshot files 
```

Comment:
why not introducing an **explicit** parameter instead of `...`, e.g.
`runSimulationsFromSnapshot <- function(snaphotPathsOrFolder, output = ".", csv = TRUE, pkml = FALSE, xml = FALSE)`

Reply:
Because this way we can pass several files or folder in one call:
```r
runSimulationsFromSnapshot("snapshot.json","snapshotFolder", exportCSV = TRUE, exportPKML = TRUE) 
```

Reply:
Note that:
```r
runSimulationsFromSnapshot(c("snapshot.json","snapshotFolder"), exportCSV = TRUE, exportPKML = TRUE)
```
Will also work

Reply:
> Note that:
> 
> ```r
> runSimulationsFromSnapshot(c("snapshot.json","snapshotFolder"), exportCSV = TRUE, exportPKML = TRUE)
> ```
> 
> Will also work

that's what I had in mind when suggesting introducing a named argument

Reply:
With `...` both work